cosmos/converter.py

```diff
@@ -225,19 +228,30 @@ def __init__(
         dbt_vars = project_config.dbt_vars or operator_args.get("vars")
 
         cache_dir = None
+        cache_identifier = None
         if settings.enable_cache:
-            cache_dir = cache._obtain_cache_dir_path(cache_identifier=cache._create_cache_identifier(dag, task_group))
+            cache_identifier = cache._create_cache_identifier(dag, task_group)
+            cache_dir = cache._obtain_cache_dir_path(cache_identifier=cache_identifier)
 
+        previous_time = time.process_time()
```

Comment:
Probably want `perf_counter()` instead?

Reply:
Thanks, @jedcunningham , I'll change it!